BoxBrowseSDKSampleApp/BoxBrowseSDKSampleApp.xcodeproj/project.pbxproj

```diff
@@ -1,510 +1,988 @@
-				PRODUCT_NAME = "$(TARGET_NAME)";
-			};
-			name = Debug;
-		};
-		15146EF81AC20C71000BA380 /* Release */ = {
-			isa = XCBuildConfiguration;
-			baseConfigurationReference = CF99696CF15113BE93F20F94 /* Pods.release.xcconfig */;
-			buildSettings = {
-				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				INFOPLIST_FILE = BoxBrowseSDKSampleApp/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_NAME = "$(TARGET_NAME)";
-			};
-			name = Release;
-		};
-		15146EFA1AC20C71000BA380 /* Debug */ = {
-			isa = XCBuildConfiguration;
-			buildSettings = {
-				BUNDLE_LOADER = "$(TEST_HOST)";
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(SDKROOT)/Developer/Library/Frameworks",
-					"$(inherited)",
-				);
-				GCC_PREPROCESSOR_DEFINITIONS = (
-					"DEBUG=1",
-					"$(inherited)",
-				);
-				INFOPLIST_FILE = BoxBrowseSDKSampleAppTests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				PRODUCT_NAME = "$(TARGET_NAME)";
-				TEST_HOST = "$(BUILT_PRODUCTS_DIR)/BoxBrowseSDKSampleApp.app/BoxBrowseSDKSampleApp";
-			};
-			name = Debug;
-		};
-		15146EFB1AC20C71000BA380 /* Release */ = {
-			isa = XCBuildConfiguration;
-			buildSettings = {
-				BUNDLE_LOADER = "$(TEST_HOST)";
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(SDKROOT)/Developer/Library/Frameworks",
-					"$(inherited)",
-				);
-				INFOPLIST_FILE = BoxBrowseSDKSampleAppTests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				PRODUCT_NAME = "$(TARGET_NAME)";
-				TEST_HOST = "$(BUILT_PRODUCTS_DIR)/BoxBrowseSDKSampleApp.app/BoxBrowseSDKSampleApp";
-			};
-			name = Release;
-		};
-/* End XCBuildConfiguration section */
-
-/* Begin XCConfigurationList section */
-		15146ECE1AC20C71000BA380 /* Build configuration list for PBXProject "BoxBrowseSDKSampleApp" */ = {
-			isa = XCConfigurationList;
-			buildConfigurations = (
-				15146EF41AC20C71000BA380 /* Debug */,
-				15146EF51AC20C71000BA380 /* Release */,
-			);
-			defaultConfigurationIsVisible = 0;
-			defaultConfigurationName = Release;
-		};
-		15146EF61AC20C71000BA380 /* Build configuration list for PBXNativeTarget "BoxBrowseSDKSampleApp" */ = {
-			isa = XCConfigurationList;
-			buildConfigurations = (
-				15146EF71AC20C71000BA380 /* Debug */,
-				15146EF81AC20C71000BA380 /* Release */,
-			);
-			defaultConfigurationIsVisible = 0;
-			defaultConfigurationName = Release;
-		};
-		15146EF91AC20C71000BA380 /* Build configuration list for PBXNativeTarget "BoxBrowseSDKSampleAppTests" */ = {
-			isa = XCConfigurationList;
-			buildConfigurations = (
-				15146EFA1AC20C71000BA380 /* Debug */,
-				15146EFB1AC20C71000BA380 /* Release */,
-			);
-			defaultConfigurationIsVisible = 0;
-			defaultConfigurationName = Release;
-		};
-/* End XCConfigurationList section */
-	};
-	rootObject = 15146ECB1AC20C71000BA380 /* Project object */;
-}
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't believe it was caused by Xcode 8, but rather just by cocoapods, which I assume decided it needed to rebuild the pbxproj file due to the addition of use_frameworks.

https://github.com/CocoaPods/CocoaPods/wiki/Generate-ASCII-format-xcodeproj 

I did confirm that the project runs fine in Xcode 7. There's a workaround described in the article above to prevent cocoapods from switching to a plist-based file. Do you think we should avoid switching to the plist format? I'm not sure what the downsides are (if any).